WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -8218,8 +8218,8 @@
 				ORGANIZATIONNAME = WordPress;
 				TargetAttributes = {
 					1D6058900D05DD3D006BFB54 = {
-						DevelopmentTeam = PZYM8XX95Q;
 						LastSwiftMigration = 1000;
+						ProvisioningStyle = Automatic;
```

Comment:
I don't think we can change to automatic provisioning without involving Platform9 😬

Reply:
mmm... oops. Not sure why it changed for me 🤔

Reply:
Xcode does as Xcode pleases 😆

Reply:
😆